modules/st2-auto-form/tests/test-array.js

```diff
@@ -30,11 +30,11 @@ describe('AutoForm ArrayField', () => {
 
     const c = new TestComponent(<ArrayField {...props} />);
 
-    c.makeChange('t,e,s,t');
+    c.makeChange('["t","e","s","t"]');
```

Comment:
now (after much discussion) we should: 
https://github.com/StackStorm/st2web/issues/394